pkg/util/test/util.go

```diff
@@ -112,6 +112,9 @@ const (
 	defaultPxAuthValidationTimeout  = 20 * time.Minute
 	defaultPxAuthValidationInterval = 30 * time.Second
 
+	defaultDeleteStorageClusterTimeout  = 30 * time.Minute
```

Comment:
do we really need 30 minutes to retry the stc deletion? it shouldn't take that long. is 10 minutes better?

Reply:
I don't know, network instabilities might cause `px-node-wiper` pods potentially can take longer to pull images, I thought its better to have a little bigger timeout than to fail due to these things. Our lab occasionally goes unstable in terms of resources ^^